BookStoreAPI/Controllers/BookStoreController.cs

```diff
@@ -0,0 +1,63 @@
+using AutoMapper;
+using Microsoft.AspNetCore.JsonPatch;
+using BookStoreAPI.Exceptions;
+using BookStoreAPI.Interfaces;
+using BookStoreAPI.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BookStoreAPI.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class BookStoreController : ControllerBase
```

Comment:
Jeśli chcesz dalej rozwijać tę aplikację to można dopisać testy integracyjne na endpointy API. https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-7.0

Reply:
W przyszłości.